src/containers/Tenant/Diagnostics/TenantOverview/TenantCpu/TopNodesByCpu.tsx

```diff
@@ -22,13 +22,14 @@ export function TopNodesByCpu({tenantName, additionalNodesProps}: TopNodesByCpuP
     const [autoRefreshInterval] = useAutoRefreshInterval();
     const columns = getTopNodesByCpuColumns(additionalNodesProps?.getNodeRef);
 
-    const {currentData, isFetching, error} = topNodesApi.useGetTopNodesQuery(
-        {tenant: tenantName, sortValue: 'CPU'},
+    const {currentData, isFetching, error} = nodesApi.useGetNodesQuery(
```

Comment:
Made all nodes tables use the same api. This way they will have single cache entry and single prepare function